### PR TITLE
Blazor Make Default Template Favicon Blazor Logo

### LINKED ISSUE #39159: Blazor Make Default Template Favicon Blazor Logo
Consider updating the templates to use the Blazor logo instead of the current default:

image

image

### PR DESCRIPTION
**Pull request by cerasamson**
# Blazor Make Default Template Favicon Blazor Logo

Updated Blazor templates to replace current favicon to Blazor logo.

**Previous favicon:**
image

**Proposed favicon (light mode):**
image

**Proposed favicon (dark mode):**
image

Fixes #39159

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=41991) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=41991) before we can accept your contribution.<br/><br/>:x: cerasamson [sign now](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=41991)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/aspnetcore?pullRequest=41991) it.</sub>

File: src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/index.html [hunk 1]
```diff
@@ -8,16 +8,17 @@
...
     <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
     <link href="css/app.css" rel="stylesheet" />
-<!--#if (Hosted)  -->
+    <!--#if (Hosted)  -->
     <link href="ComponentsWebAssembly-CSharp.Client.styles.css" rel="stylesheet" />
-<!--#else  -->
+    <!--#else  -->
     <link href="ComponentsWebAssembly-CSharp.styles.css" rel="stylesheet" />
-<!--#endif  -->
+    <!--#endif  -->
```
Comment (TanayParikh) [inline src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/index.html:15]: Could we revert these changes, I believe the editor may've just done these on it's own.

File: src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/index.html [hunk 2]
```diff
@@ -8,16 +8,17 @@
...
     <!--#if PWA -->
     <link href="manifest.json" rel="manifest" />
     <link rel="apple-touch-icon" sizes="512x512" href="icon-512.png" />
     <link rel="apple-touch-icon" sizes="192x192" href="icon-192.png" />
     <!--#endif -->
+    <link rel="shortcut icon" type="image/png" href="favicon.png" />
```
Comment (TanayParikh) [inline src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/index.html:21]: Could we please move this above line 11, along with the other links?

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/_Host.cshtml [hunk 3]
```diff
@@ -13,6 +13,7 @@
     <link href="css/site.css" rel="stylesheet" />
     <link href="BlazorServerWeb-CSharp.styles.css" rel="stylesheet" />
     <component type="typeof(HeadOutlet)" render-mode="ServerPrerendered" />
+    <link rel="shortcut icon" type="image/png" href="favicon.png"/>
```
Comment (TanayParikh) [inline src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/_Host.cshtml:16]: Could we please move this above line 11, along with the other links?

Out of curiosity, does the `favicon.png` work without this `link` attribute? We previously didn't need it because we were just using the `favicon.ico`.

Comment (TanayParikh) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/_Host.cshtml:16]: Hmm looks like we may be able to remove `shortcut` from here as it's an artifact of IE and [IE isn't officially supported by Blazor](https://docs.microsoft.com/en-us/aspnet/core/blazor/supported-platforms?view=aspnetcore-6.0).

https://www.kevinleary.net/cross-browser-favicons/

**Review by TanayParikh — APPROVED** [review]
This looks great! 🎉

**Review by TanayParikh — APPROVED** [review]
